diff --git a/node_modules/onnxruntime-react-native/android/build.gradle b/node_modules/onnxruntime-react-native/android/build.gradle
index 41b4359..f24db9c 100644
--- a/node_modules/onnxruntime-react-native/android/build.gradle
+++ b/node_modules/onnxruntime-react-native/android/build.gradle
@@ -247,7 +247,8 @@ dependencies {
     extractLibs "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
   }
 
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
+  def reactNativeMajorMinor = REACT_NATIVE_VERSION.tokenize('.').take(2).collect { it.toInteger() }
+  if (reactNativeMajorMinor[0] < 0 || (reactNativeMajorMinor[0] == 0 && reactNativeMajorMinor[1] < 71)) {
     extractLibs "com.facebook.fbjni:fbjni:+:headers"
     extractLibs "com.facebook.fbjni:fbjni:+"
   }
